climada/engine/impact.py

```diff
@@ -1194,6 +1195,11 @@ def plot_rp_imp(
             plot impact as log10(impact). Default: True
         smooth : bool, optional
             smooth plot to plot.RESOLUTIONxplot.RESOLUTION. Default: True
+        mask_relative_distance: float, optional
+            Relative distance (with respect to maximal map extent in longitude or latitude) to data
+            points above which plot should not display values. For instance, to only plot values
+            at the centroids, use mask_relative_distance=0.01. If None, the plot is not masked.
+            Default is None.
```

Comment:
🧐 This parameter is really hard to describe. I've read it repeatedly and still need the example images to understand it. 
For me it would be probably easier if it was written in a more "positive" way, something like

```suggestion
        mask_relative_distance: float, optional
            Only regions are plotted that are closer to any of the data points than this distance,
            relative to overall plot size. For instance, to only plot values
            at the centroids, use mask_relative_distance=0.01. If None, the plot is not masked.
            Default is None.
```
Though I'm not sure, whether it's better like that.

Then: don't meant to be fuzzy but the parameter name is a somewhat awkward, don't you think?
Is it not rather `relative_mask_distance`? Or just `mask_distance`? Or what about `plotting_range`? (as in only plot stuff within this plotting range)

Reply:
agreed, I will change the description and the parameter name to `mask_distance`. Thanks!